### DISCUSSION TITLE
Video Won't Play

### DISCUSSION THREAD

**Discussion by radrogue1** [jspsych/jsPsych#1874] [Q&A] [answered]
For my code, I want to play a video using the video-button-response plugin. This is the way that I am calling the plugin:

`var video_instructions = {
        type: 'video-button-response',
        stimulus: "/static/video/instructional_video.mp4",
        choices: ["Yes", "No"],
        prompt: 'Did you understand the instructions?',
        response_allowed_while_playing: false,
        autoplay: true,
    }`

While the buttons appear, the video does not at all, even though the code claims to have preloaded the video. When I inspect the element, and look specifically at the video this is what the html output of this code is:
<img width="435" alt="Screen Shot 2021-06-02 at 2 43 10 PM" src="https://user-images.githubusercontent.com/68128449/120535330-dc24ed00-c3b0-11eb-9804-d3c6093af708.png">
I would really appreciate it if someone could help me with this issue! Thanks

**Comment by jodeleeuw** [comment, marked as answer]
Hi @radrogue1 ,

The `stimulus` parameter for the video plugins should be an array ([see docs here](https://www.jspsych.org/plugins/jspsych-video-button-response/)). This is because `<video>` elements in HTML allow for multiple source files to be specified so that if one kind of video files type can't be played on a particular browser a different one can be used instead.
